Resources/Prototypes/_CD/Entities/Objects/Misc/lunchboxes.yml

```diff
@@ -0,0 +1,56 @@
+- type: entity
+  id: LunchboxBase
+  parent: BaseStorageItem
+  abstract: true
+  components:
+  - type: Sprite
+    layers:
+    - map: [ base ]
+      state: icon
+  - type: EmitSoundOnLand
+    sound:
+      path: /Audio/Items/toolbox_drop.ogg
+  - type: Storage
+    capacity: 25 # ffs I forgot CD uses list inv... I hope this is enough for a sandwich and a drink
+  - type: Item
+    size: 30
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 3
+    soundHit:
+      path: "/Audio/Weapons/smash.ogg"
+  - type: Tag
+    tags:
+    - Toolbox
```

Comment:
Is there any reason for the toolbox tag? From the looks of things it is only used in cargo bounties (unless I missed something).

Reply:
This is what happens when I copy and paste code at 11pm

No real reason for it aside from me just forgetting to review it after I took it from the original toolbox thingy. I'll take it out in a few hours as I am going to bed

Reply:
No worries, get your sleep.